localstripe/server.py

```diff
@@ -265,8 +265,9 @@ async def f(request):
 
 
 for cls in (Charge, Coupon, Customer, Event, Invoice, InvoiceItem,
-            PaymentIntent, PaymentMethod, Plan, Product, Refund, SetupIntent,
-            Source, Subscription, SubscriptionItem, TaxRate, Token):
+            PaymentIntent, PaymentMethod, Payout, Plan, Product, Refund,
+            SetupIntent, Source, Subscription, SubscriptionItem,
+            TaxRate, Token,):
```

Comment:
To keep the same coding style:
```suggestion
            SetupIntent, Source, Subscription, SubscriptionItem, TaxRate,
            Token):
```